pkg/auth/rbac.go

```diff
@@ -146,6 +146,11 @@ func (r *Rbac) Run(ctx context.Context, mapOfTuple, mapOfAuth *ebpf.Map) {
 				continue
 			}
 
+			if msgType == MSG_TYPE_IPV6 {
```

Comment:
why do we need this, since we have converted in sockips

```
    if (is_ipv4_mapped_addr(skops->local_ip6)) {
        (*msg).type = IPV4;
    }
```